### PR TITLE
Update about page content

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
- Update about page content
- Apply design

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/about
- Ensure the copy matches [the copy doc](https://docs.google.com/document/d/1UM4K9n9SFNiWdmKIBnPGCsjLQrPSOnqZHMNIMwQ2CZI/edit#)
- Check the page looks like [the design](https://app.zeplin.io/project/5eb3d16a91c4117789940141/screen/5ee2140d0dbd73b3d0f5e17d)

### CODE REVIEW

File: templates/appliance/about.html [hunk 1]
```diff
@@ -8,146 +8,198 @@
...
   </div>
 </section>
 
 <section class="p-strip">
-  <div class="row">
-    <div class="col-8">
-      <h2>Built on trusted infrastructure</h2>
-      <p>All Ubuntu Appliance images and open source components are assembled on the same secure infrastructure as Ubuntu. Proprietary components are built by the official vendor for that application, just like the software you download directly from them. Since an Ubuntu Appliance is made of digitally signed packages, you always know exactly where each component comes from, when it was updated, and which versions are available.</p>
+  <div class="row u-equal-height">
+    <div class="col-4 u-vertically-center u-align--center u-hide--small">
```
Comment (pmahnke) [inline templates/appliance/about.html:118]: I think the next bit is col-6 by col-6

File: templates/appliance/about.html [hunk 2]
```diff
@@ -8,146 +8,198 @@
...
-    </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
   </div>
-</section>
 
-<section class="p-strip--light">
-  <div class="row">
+  <div class="row u-equal-height">
     <div class="col-8">
```
Comment (pmahnke) [inline templates/appliance/about.html:141]: col-6

File: templates/appliance/about.html (hunk 1 above)
Comment (anthonydillon) [reply templates/appliance/about.html]: done

File: templates/appliance/about.html (hunk 2 above)
Comment (anthonydillon) [reply templates/appliance/about.html]: done

**Pull request merged**